sdk/documenttranslation/Azure.AI.DocumentTranslation/src/DocumentStatusDetail.cs

```diff
@@ -31,5 +31,11 @@ public partial class DocumentStatusDetail
         /// <summary> Date time in which the Document's status has been updated. </summary>
         [CodeGenMember("LastActionDateTimeUtc")]
         public DateTimeOffset LastModified { get; }
+
+        /// <summary> Returns true if the document translation is completed. </summary>
```

Comment:
should we add something in the comment that this does not guarantee the state of the document? i.e. failed or succeeded?

Reply:
@maririos something like this?

> Returns true if the document translation is completed successfully or otherwise.

Reply:
what do you think about `Returns true if the translation on the document is completed, independent if it succeeded or failed`.